packages/yarnpkg-core/sources/Cache.ts

```diff
@@ -12,7 +12,7 @@ import * as miscUtils                                                   from './
 import * as structUtils                                                 from './structUtils';
 import {LocatorHash, Locator}                                           from './types';
 
-const CACHE_VERSION = 8;
+const CACHE_VERSION = 9;
```

Comment:
Updating the global cache version for this isn't something we should do, instead we should add a cache version to git dependencies so we only need to regenerate those.
Something like what we do for patch dependencies https://github.com/yarnpkg/berry/blob/7ae458b8165ad53a8ef9db0060cbb6de73305768/packages/plugin-patch/sources/PatchResolver.ts#L9